platforms/enterprise/enterprise-plugin-performance/build.gradle.kts

```diff
@@ -59,11 +59,16 @@ subprojects {
 }
 
 tasks.withType<gradlebuild.performance.tasks.PerformanceTest>().configureEach {
-    systemProperties["incomingArtifactDir"] = "$rootDir/incoming/"
-
     environment("GRADLE_INTERNAL_REPO_URL", System.getenv("GRADLE_INTERNAL_REPO_URL"))
     environment("GRADLE_INTERNAL_REPO_USERNAME", System.getenv("GRADLE_INTERNAL_REPO_USERNAME"))
     environment("GRADLE_INTERNAL_REPO_PASSWORD", System.getenv("GRADLE_INTERNAL_REPO_PASSWORD"))
 
     reportGeneratorClass = "org.gradle.performance.results.BuildScanReportGenerator"
+
+    // Property required by `AbstractBuildScanPluginPerformanceTest`
+    val pluginInfoDirProperty = "org.gradle.performance.enterprise.plugin.infoDir"
+    val pluginInfoDir = System.getProperty(pluginInfoDirProperty) ?: rootDir.resolve("incoming").path
```

Comment:
🤔 I think we have been using different properties for the command line and the system property for the test JVM. I think it would make sense to use a project property to configure the location from the command line, while using a system property on the test JVM. Something like `-PenterprisePluginInfoDir=...` should be fine, WDYT?

Reply:
I don't have a preference either way. Before you mentioned, that we usually pass TC props as system props to the build. What made you change your mind and suggest a project property?

Reply:
I think I erred before: We pass TC settings as project properties to the build, and then the build passes the project properties as system properties to the test JVM. Looking at the current command line it seems to be a mix, though I think I'd prefer the project properties, like for example `-PtestJavaVersion=19 -PtestJavaVendor=openjdk` that we already have.

Reply:
Done